suite-native/storage/src/tests/migrations/deviceV2.test.ts

```diff
@@ -0,0 +1,63 @@
+import { migrateDeviceState } from '../../migrations/device/v2';
+
+describe('migrateDeviceState', () => {
+    it('should migrate old devices without state to new format', () => {
+        const oldDevices = [
+            {
+                state: undefined,
+                otherKey: 'otherValue',
+            },
+        ];
+
+        const migratedAccounts = migrateDeviceState(oldDevices);
```

Comment:
why the name migratedAccounts in the whole file?

Reply:
Copied from another test and forgot to change it 😄